Lib/test/test_argparse/test_translations.py

```diff
@@ -0,0 +1,60 @@
+import argparse
```

Comment:
Does this need to be in a new file? Could we not just add these tests to the existing test_argparse.py file?